sdk/search/search-documents/test/utils/setupIndex.ts

```diff
@@ -534,3 +534,60 @@ export async function deleteIndexers(client: SearchIndexerClient): Promise<void>
     await client.deleteIndexer(`my-azure-indexer-${i}`);
   }
 }
+
+export async function createSynonymMaps(client: SearchIndexClient): Promise<void> {
+  for (let i = 1; i <= 4; i++) {
+    await client.createSynonymMap({
+      name: `my-azure-synonymmap-${i}`,
+      synonyms: ["United States, United States of America => USA", "Washington, Wash. => WA"]
+    });
+  }
+}
+
+export async function deleteSynonymMaps(client: SearchIndexClient): Promise<void> {
+  for (let i = 1; i <= 4; i++) {
+    await client.deleteSynonymMap(`my-azure-synonymmap-${i}`);
+  }
+}
+
+export async function createSimpleIndex(client: SearchIndexClient, name: string): Promise<void> {
```

Comment:
The existing index is little complex and to modify it (for the last test case) seems to be next to impossible. Whatever I try, the service is returning back with Cannot be modified. So, I preferred to have a simple index to deal with the test cases related to indexes.

Reply:
That sounds reasonable to me.